deltacat/aws/clients.py

```diff
@@ -46,7 +46,7 @@ class RetryIfRetryableHTTPStatusCode(retry_if_exception):
 
     def __init__(self):
         def is_retryable_error(exception):
-            return (
+            return isinstance(exception, requests.exceptions.ConnectionError) or (
```

Comment:
Are you sure it captures all errors we saw in production?

Reply:
We were encountering`HttpError` in production, so it is a safe bet for the time being. Adding `ConnectionError` is more of a precautionary step after looking at the request [docs](https://requests.readthedocs.io/en/latest/user/quickstart/?highlight=exceptions#errors-and-exceptions), but going forward, we can incrementally add new exceptions here as needed.